SparkPost/support-docs
# WE-338 Replace logo with new logo

**What Changed**
- Adds new Messagebird logo

**How to verify**
- Run the site or visit the deploy preview
- Verify the logo
